doc/man1/srec_cat.1

```diff
@@ -356,8 +356,12 @@ write the file.  See \f[I]srec_needham\fP(5) for a description of this
 file format.
 .\" -----   --output -O*   -------------------------------------------------
 .TP 8n
-\fB\-Ohio_Scientific\fP
-This option says to use the Ohio Scientific hexadecimal format.
+\fB\-Ohio_Scientific65A\fP
+This option says to use the Ohio Scientific 65A hexadecimal format.
+See \f[I]srec_os65a\fP(5) for a description of this format.
+.TP 8n
+\fB\-Ohio_Scientific65V\fP
```

Comment:
Suggest inserting immediately above this line:
`\fB\-Ohio_Scientific\fP`

This shows both the old and the new command will give os65v.